examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -21,11 +21,12 @@
      "name": "stdout",
```

Comment:
<p>over here, I also get this notif:</p><p> </p><pre class="ql-syntax" spellcheck="false">The number of effective samples is smaller than 10% for some parameters.
</pre><p> </p><p>I checked <a href="https://discourse.pymc.io/t/the-number-of-effective-samples-is-smaller-than-25-for-some-parameters/1050/3" target="_blank">this page</a> for a similar thing, should we be addressing this? mentioning something about what to do if a similar thing comes up?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
No need to worry about that here